kernels/volk/volk_32fc_s32f_power_spectrum_32f.h

```diff
@@ -128,7 +128,10 @@ volk_32fc_s32f_power_spectrum_32f_a_sse3(float* logPowerOutput, const lv_32fc_t*
     const float real = *inputPtr++ * iNormalizationFactor;
     const float imag = *inputPtr++ * iNormalizationFactor;
 
-    *destPtr = 10.0*log10f(((real * real) + (imag * imag)) + 1e-20);
+    // *destPtr = 10.0*log10f(((real * real) + (imag * imag)) + 1e-20);
+    // For performance rewrite log10 to use log2
+    *destPtr = volk_log2to10factor *
+               log2f_non_ieee(((real * real) + (imag * imag)) + 1e-20);
```

Comment:
The `+ 1e-20` should no longer be necessary, because `log2f_non_ieee` already returns -127 instead of -Inf when the input is zero.

Reply:
There are a bunch more places below where `+ 1e-20` is used and I think they can all be removed.